src/binary-reversal/index.js

```diff
@@ -3,6 +3,47 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  let stringToNumber = parseInt(value);
+  let decimalToBinary = stringToNumber.toString(2);
+
+  if (decimalToBinary.length < 8) {
+    let paddedBinary = "0000000" + stringToNumber.toString(2);
+    let actualBinary = paddedBinary.substring(
+      paddedBinary.length - 8,
+      paddedBinary.length
+    );
+
+    let splitted = actualBinary.split("");
+    let reversedSplittedString = splitted.reverse();
+    let joinedReversedSplittedString = reversedSplittedString.join("");
+
+    let binaryLength = joinedReversedSplittedString.length;
+    var sum = 0;
+
+    for (let i = 0, fac = 1; i < binaryLength; i++, fac *= 2)
+      sum += fac * joinedReversedSplittedString[binaryLength - i - 1];
+
+    return sum.toString();
+  }
+
+  let nonPaddedBinary = stringToNumber.toString(2);
```

Comment:
I think the entire block below could've been avoided and you could've done a single pass through the string.